README.md

```diff
@@ -28,24 +30,24 @@ It is assumed that ROS is installed.
 
 1. Install the QP solver you wish to use according to [this section](https://github.com/isri-aist/QpSolverCollection#qp-solver-installation). You can skip installing QP solvers that you do not use.
 
-2. Setup catkin workspace.
+2. Setup colcon workspace.
 ```bash
 $ mkdir -p ~/ros/ws_qp_solver_collection/src
-$ cd ~/ros/ws_qp_solver_collection
-$ wstool init src
```

Comment:
I think the wstool commands can be kept for installing dependent repositories from source.